docs/runtimes/image-build.md

```diff
@@ -107,12 +107,12 @@ project.build_function("trainer", with_mlrun=True, mlrun_version_specifier="1.0.
 ```
 
 ### Working with code repository
-As the code matures and evolves, the code will usually be stored in a git code repository.
+As the code matures and evolves, the code is usually stored in a git code repository.
```

Comment:
maybe worth mentioning that it can be stored in archive file like: tar.gz or zip